exercises/L-functions-nested/exercise.js

```diff
@@ -3,3 +3,23 @@ var mentor2 = "Irina";
 var mentor3 = "Mimi";
 var mentor4 = "Rob";
 var mentor5 = "Yohannes";
+
+
+function shoutyGreeting(name){
```

Comment:
This exercise had many trainees get to use more of their brains. There are many solutions to any code. Your basic functions show great understanding on Declaring Functions and returning them. Well done!